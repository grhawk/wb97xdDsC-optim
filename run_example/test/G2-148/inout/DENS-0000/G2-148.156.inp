 $BASIS NGAUSS=6 GBASIS=N31 $END
 $SYSTEM MWORDS=8 $END
 $SCF DIRSCF=.t. $END
 $CONTRL SCFTYP=UHF RUNTYP=ENERGY DFTTYP=wB97X EXETYP=RUN MAXIT=200 
ICHARG=0 MULT=3 $END
 $DATA
 Should be setted!
 C1
 O     8.00     0.000000     0.000000     0.000000
 $END
 $DFT NRAD=24 DDSC=.t. NLEB=110 $END