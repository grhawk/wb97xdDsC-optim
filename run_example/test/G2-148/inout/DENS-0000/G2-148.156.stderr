/var/spool/slurm/job12429/slurm_script: line 9: module: command not found
